### ISSUE TITLE
Set-ExecutionPolicy RemoteSigned is not recommended

### ISSUE THREAD

**Issue by patrix87** [alekdavis/PlexBackup#20]
Instead you can simply run as admin a .cmd file with the following line in it : 

```batch
start powershell.exe -noprofile -executionpolicy bypass -file .\PlexBackup.ps1 -ConfigFile ".\PlexBackup.ps1.json"
```

This way you don't permanantly change security settings on you system.

Also you can adapt the following function to your code in order to require admin when the script start 

```ps
Function RequireAdmin {
	If (!([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]"Administrator")) {
		Start-Process powershell.exe "-NoProfile -ExecutionPolicy Bypass -File `"$PSCommandPath`" $PSCommandArgs" -Verb RunAs
		Exit
	}
}
```

**Comment by alekdavis** [comment]
For the first point, it's totally up to the user how to set up or bypass the policy. I specified the common recommended approach, but if the user feels it's inadequate, nothing stops them to use an alternative. I do not want to expand the instructions into the tutorial of how to or not to use PowerShell.

For the second suggestion, wouldn't the invocating user be prompted to agree with elevated security context? I did not try it, but I assume this is what should happen (otherwise, any malicious script would do it behind the scene without the user knowing). So it would not be totally seamless, and I'd rather know what the users are doing in advance.
